### ISSUE TITLE
Navbar dropdown menus are not accessible

### ISSUE THREAD

**Issue by robertknight** [hypothesis/h#4555]
The dropdown menus in the navbar as seen at eg. https://hypothes.is/search are not accessible. When testing with VoiceOver in OS X 10.12 I encountered the following issues:

1. The "Settings" menu link is read as "down pointing small triangle"
1. There is no indication in VoiceOver that the Groups or Settings menus are popup menu triggers
1. When the menu is opened, the contents of the menu are not automatically focused, so navigating to the menu is difficult
1. Navigating the contents of the Groups and Settings menus does not announce the contents of the popup as a menu item

For examples of accessible dropdown menus, see the navbars in apps such as Gmail & Facebook.
